yiisoft/yii2/CVE-2024-4990.yaml

```diff
@@ -0,0 +1,8 @@
+title: Unsafe Reflection in base Component class
+link: https://github.com/yiisoft/yii2/security/advisories/GHSA-cjcc-p67m-7qxm
+cve: CVE-2024-4990
+branches:
+    2.0.49.x:
+        time: null
+        versions: ['<=2.0.49']
```

Comment:
This is very confusing, the github security advisory says it affects "<2.0.49" but patched version is "2.0.50", does it affect "2.0.49" or not?

Reply:
@DBX12 can you apply that change then?

Reply:
Yet another update of the advisory, so I've updated my PR.